app/workers/functions.php

```diff
@@ -101,6 +101,10 @@ class FunctionsV1 extends Worker
 
     public array $allowed = [];
 
+    public function getWorkerName(): string {
```

Comment:
```suggestion
    public function getName(): string {
```
The word `worker` is already obvious from the context. Not need to mention it again.

Reply:
Renamed all `getWorkerName` to `getName` - in abstract class, in every worker and in log where I used this method.